dnscrypt-proxy/plugin_dns64.go

```diff
@@ -93,6 +93,7 @@ func (plugin *PluginDNS64) Eval(pluginsState *PluginsState, msg *dns.Msg) error
 	if !plugin.proxy.clientsCountInc() {
```

Comment:
Step into this. The counter will not increase, if reaches the up bound. AI without review?

Reply:
I don't see how the upper bound is relevant here. It's about processIncomingQuery and how if a panic occurs somewhere in that, the counter is never from deincremented. If that happens over and over, the server will be inoperable after the never-deincremented variable reaches the limit which this like is about.